tests/unit/test_fact.py

```diff
@@ -187,7 +186,7 @@ def test_fact_edited(test_fact_empty_fx):
     Test that edited generates a datetime automagically, and returns it.
     """
     test_fact = test_fact_empty_fx
-    test_fact.edited = datetime.datetime.now(tz=timezone.utc)
+    test_fact.edited = pendulum.now()
```

Comment:
[style] We should be consistent, is it pendulum.now() or pendulum.now(pendulum.tz.UTC). Let's try to also keep this within the tests :)